### PR TITLE
Refactor exhook for HOCON schema

### PR DESCRIPTION
**Pull request by HJianBo**

### CODE REVIEW

**Comment by zmstone** [comment]
rebase to tidy up the commits ?

File: apps/emqx_exhook/src/emqx_exhook.appup.src [hunk 1]
```diff
@@ -0,0 +1,33 @@
+%% -*-: erlang -*-
```
Comment (tigercl) [inline apps/emqx_exhook/src/emqx_exhook.appup.src:1]: Remove this file?

**Review by terry-xiaoyu — DISMISSED** [review]

File: apps/emqx_exhook/src/emqx_exhook.appup.src (hunk 1 above)
Comment (HJianBo) [reply apps/emqx_exhook/src/emqx_exhook.appup.src:1]: My bad. I will empty it

**Review by tigercl — DISMISSED** [review]

File: apps/emqx_exhook/src/emqx_exhook_app.erl [hunk 2]
```diff
@@ -18,7 +18,7 @@
 
 -behaviour(application).
 
--include("src/exhook/include/emqx_exhook.hrl").
+-include("emqx_exhook.hrl").
 
 -emqx_plugin(extension).
```
Comment (turtleDeng) [inline apps/emqx_exhook/src/emqx_exhook_app.erl:23]: delete -emqx_plugin(extension).

Comment (HJianBo) [reply apps/emqx_exhook/src/emqx_exhook_app.erl]: resolved

**Review by terry-xiaoyu — APPROVED** [review]

**Pull request merged**